modules/ROOT/pages/kubernetes/quickstart-analytics-cluster.adoc

```diff
@@ -0,0 +1,168 @@
+:description: How to deploy a Neo4j cluster to a cloud or a local Kubernetes cluster using Neo4j Helm chart.
+[role=enterprise-edition]
+[[quick-start-analytic-cluster]]
+= Quickstart: Deploy a Neo4j cluster for analytic queries
+
+_The feature is available in the Neo4j Helm chart from version 5.14._
+
+This quickstart shows how to configure and deploy a special Neo4j cluster without fault tolerance to support analytic queries.
```

Comment:
use of words like "without fault tolerance" is fine but we should mention that this scenario is specifically and only for "1 primary and N secondary" ... without fault tolerance does imply that but not everyone will understand that hence if we can mention its "1 primary and N secondary" that would be great

Reply:
This decision was taken due to my intention above.